updatemodules/partitions/utils/efi/efi.go

```diff
@@ -716,7 +716,7 @@ func parseMediaType(subType uint8, data []byte) (dp interface{}, err error) {
 
 		return hd, nil
 	default:
-		return nil, nil
+		return nil, aoserrors.Errorf("unknown media type: %d", subType)
```

Comment:
The idea is: we parse only known fields, we should not return error for unknown types. Otherwise it breaks functionality,

Reply:
fixed